### ISSUE TITLE
Animation name causes "known-properties" warning if name contains the underscore character (_)

### ISSUE THREAD

**Issue by gamtiq** [CSSLint/csslint#450]
Hello,

The following code causes the warning about unexpected property value: `Expected (none | <ident>) but found 'a-move_vert'`
```css
.move {
    animation-name: a-move_vert;
}
```

**Comment by frvge** [comment]
Awaiting new version: Duplicates #514
